tests/Integrations/Custom/Autoloaded/GitMetadataTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace DDTrace\Tests\Integrations\Custom\Autoloaded;
+
+use DDTrace\Tests\Common\WebFrameworkTestCase;
+use DDTrace\Tests\Frameworks\Util\Request\GetSpec;
+
+final class GitMetadataTest extends WebFrameworkTestCase
+{
+    protected static function getAppIndexScript()
+    {
+        return __DIR__ . '/../../../Frameworks/Custom/Version_Autoloaded/public/index.php';
+    }
+
+    protected static function getEnvs()
+    {
+        return array_merge(parent::getEnvs(), [
+            'DD_TRACE_GIT_METADATA_ENABLED' => 'true',
```

Comment:
Can you please extend this test to test re-loading of the .git metadata? I.e. put a fake `.git` dir in the folder of the script during the testing, change the HEAD and observe whether the changes are picked up?